dlp/snippets/inspect_content.py

```diff
@@ -983,6 +985,67 @@ def inspect_image_file_all_infotypes(
 # [END dlp_inspect_image_all_infotypes]
 
 
+# [START dlp_inspect_image_listed_infotypes]
+def inspect_image_file_listed_infotypes(
+    project: str,
+    filename: str,
+    info_types: List[str],
+    include_quote=True,
+) -> None:
+    """Uses the Data Loss Prevention API to analyze strings for protected data
+    in image file for given info types.
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        filename: The path to the file to inspect.
+        info_types:
+        include_quote: Boolean for whether to display a quote of the detected
+            information in the results.
+    """
+    # Import the client library
+    import google.cloud.dlp
+
+    # Instantiate a client.
+    dlp = google.cloud.dlp_v2.DlpServiceClient()
+
+    # Prepare info_types by converting the list of strings into a list of
+    # dictionaries (protos are also accepted).
```

Comment:
"protos are also accepted" doesn't seem to match the implementation. If we detect a List[Proto] then the list should be unchanged, not converted. It's probably simpler to just remove the comment, but you can also change the code to detect whether the input is strings or objects.

Reply:
updated, Thank you for reviewing